README.rst

```diff
@@ -132,4 +132,9 @@ Here is example of a filter with a nested function (using PostGIS functions):
 
 ``[{"func": "st_intersects", "param": [{"func": "st_setsrid", "param": [{"func": "st_makepoint", "param": [-72, 42.36]}, 4326]}, {"func": "st_transform", "param": [{"field": "geom"}, 4326]}], "operator": "is", "value": true}]``
 
+SQLite on Files Stored in Girder
+--------------------------------
 
+SQLite works with files rather than a database server.  To specify a file that is in Girder's local file system, use a Database URI of ``sqlite:////<absolute path to file>``.  Note that four slashes are necessary (e.g., ``sqlite:////home/pliny/natural_history/zoology.db``).
```

Comment:
I could see why 3 slashes are necessary, but why 4?

Reply:
It is how sqlalchemy's sqlite handles file paths (http://docs.sqlalchemy.org/en/latest/dialects/sqlite.html#dialect-sqlite-pysqlite-url).  Specifically, the `sqlite:///` says to use sqlite, and then the remainder is the *relative* path.  For an absolute path, you therefore need to start that remainder with the *fourth* slash.

Since we will probably always demand an absolute path or a Girder resource path, commit https://github.com/OpenGeoscience/database_assetstore/pull/40/commits/9225c0765f6b2d4d993fdbc237520997c42e1b1a made it so that *two or more* slashes are required.  I can adjust the documentation to show `sqlite://<absolute path to file>`.

Reply:
ok, works for me.